user-docs/latest/content-manager/introduction-to-content-manager.md

```diff
@@ -64,14 +68,14 @@ To temporarily configure the fields displayed in the table:
 
 1. Click on the settings button <Fa-Cog />.
 2. In the Displayed Fields section, tick the boxes associated with the field you want to be displayed in the table.
-3. Untick the boxes associated with the fields you want to remove from the table.
+3. Untick the boxes associated with the fields you do not want to be displayed in the table.
 
 ::: tip NOTE
```

Comment:
I would love @derrickmehaffy validation on the following bullet points.
To make sure that makes sense.
This is a really tricky thing to explain properly and my English is not enough good to validate here.

Reply:
Looks fine to me, both would work with the new one being more formal. LGTM